src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -777,18 +778,30 @@ uct_ib_mlx5_devx_mem_reg_gva(uct_md_h uct_md, uct_mem_h *memh_p)
     }
 
     memh->super.flags |= UCT_IB_MEM_FLAG_ODP;
-    status             = uct_ib_reg_mr(&md->super, NULL, SIZE_MAX, &params,
-                                       UCT_IB_MEM_ACCESS_FLAGS | IBV_ACCESS_ON_DEMAND, NULL,
-                                       &memh->mrs[UCT_IB_MR_DEFAULT].super.ib);
+    access_flags       = uct_ib_memh_access_flags(&md->super, &memh->super);
```

Comment:
can we unite common code with uct_ib_verbs_mem_reg?
the overall structure of the function looks very similar, except here we need to register with ODP and pass specific address/length

Reply:
main difference is - different layout of memh